public/app/plugins/datasource/cloud-monitoring/components/SLO/LookbackPeriodSelect.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { FunctionComponent } from 'react';
+
+import { SelectableValue } from '@grafana/data';
+import { Select } from '@grafana/ui';
+
+import { QueryEditorRow } from '..';
+import { SELECT_WIDTH, LOOKBACK_PERIODS } from '../../constants';
+
+export interface Props {
+  refId: string;
+  onChange: (lookbackPeriod: string) => void;
+  selectorName: string;
+  templateVariableOptions: Array<SelectableValue<string>>;
+  current?: string;
+}
+
+export const LookbackPeriodSelect: FunctionComponent<Props> = ({
+  refId,
+  selectorName,
+  current,
+  templateVariableOptions,
+  onChange,
+}) => {
+  if (selectorName === 'select_slo_burn_rate') {
```

Comment:
nit: Rather than adding an `if` block for the whole component, in the parent component (SLOQueryEditor), you can do:

```tsx
{query.selectorName === 'select_slo_burn_rate'  && <LookbackPeriodSelect ... />}
```

And assume here that it's for that case.

Reply:
Indeed, fixed https://github.com/grafana/grafana/pull/53710/commits/9a3d0db112b81432c666bcdd9d9f621f108cabfc